packages/xivlauncher-cn-git/xivlauncher-cn-git.pacscript

```diff
@@ -0,0 +1,32 @@
+name="xivlauncher-cn-git"
+pkgname="xivlauncher-cn"
+gives="xivlauncher-cn"
+repology=("project: xivlauncher-cn")
+version="1.0.2"
+url="https://github.com/ottercorp/XIVLauncher.Core.git"
+license="GPL-3.0"
+maintainer="qy117121 <chuan@ubuntu.org.cn>"
+build_depends="dotnet-sdk-6.0 git"
+depends="build-essential aria2 libsdl2-2.0-0 libsecret-1-0 attr fontconfig liblcms2-2 libxml2 libxcursor1 libxrandr2 libxdamage1 libxio0 gettext libfreetype6 libglu1-mesa libsm6 libpcap0.8 libfaudio0 desktop-file-utils libjxr0 xdg-utils "
```

Comment:
`build-essential` is already installed and even then, should not be in `depends`